samples/cxx/SConscript

```diff
@@ -90,12 +94,20 @@ set(CMAKE_EXE_LINKER_FLAGS ${CMAKE_EXE_LINKER_FLAGS} ${OpenMP_EXE_LINKER_FLAGS})
         env_args.append('MSVC_VERSION={0!r}'.format(localenv['MSVC_VERSION']))
     localenv['tmpl_env_args'] = ', '.join(env_args)
 
-    # If this is a conda build on macOS, we do not want to specify the conda
-    # compilers from the build environment, because those won't be installed
-    # on the user's system.
-    if env['OS'] == 'Darwin' and os.environ.get('CONDA_BUILD', False):
-        localenv['CXX'] = 'clang++'
-        localenv['CC'] = 'clang'
+    if localenv["package_build"]:
+        # We do not want to specify the conda compilers from the build environment,
+        # because those won't be installed on the user's system.
+        if localenv["OS"] == "Darwin":
+            localenv["CXX"] = "clang++"
+            localenv["CC"] = "clang"
+        elif env["OS"] == "Windows":
+            # compilation needs the Visual Studio Command Prompt
+            # TODO: verify
+            localenv["CXX"] = ""
+            localenv["CC"] = ""
```

Comment:
I would assume that the C/C++ compiler should be set to `cl` to use VS. I'm not sure what the best default guess would be for a Fortran compiler on Windows. Maybe `ifort` or `ifx` are the most likely to be available?

Reply:
I don’t think that Fortran can be assumed to be available on Windows?